controllers/apply-next/form-router.js

```diff
@@ -196,29 +226,92 @@ function initFormRouter(formModel) {
         next();
     });
 
+    // Require login before using everything else after this
+    router.use((req, res, next) => {
```

Comment:
This should probably be moved up above (or in to?) the previous `req.use`. Currently it looks like we're doing a bunch of work in the previous middleware that might not apply if the user isn't logged in yet.